composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name":"yubico/u2flib-server",
+  "name":"ProtonMail/u2flib-server",
```

Comment:
The idea is merge this in the yubico branch, we should keep it the same for now. If yubico doesn't accept PRs we will change it later.